online_meetings/report.py

```diff
@@ -21,6 +21,26 @@
 DEFAULT_SLEEP_TIME = ENV.get('DEFAULT_SLEEP_TIME', 10)
 
 
+def to_seconds(duration: str) -> int:
+    """Returns a value of a string duration in seconds
+       This string is expected in HH:MM:SS or MM:SS or SS
+
+    :param timestr: String in a duration format like HH:MM:SS
+    :type timestr: str
+    :return: Value in seconds
+    :rtype: int
+    """
+    total_seconds = 0
+    for second in duration.split(':'):
+        try:
+            total_seconds = total_seconds * 60 + int(second)
+        # If there is an error just return 0
+        except ValueError:
+            logger.warn(f"{str} is not a valid duration")
```

Comment:
Looks like `warn` is deprecated: https://docs.python.org/3/library/logging.html#logging.Logger.warning

Reply:
Always forget this!